website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -357,7 +357,7 @@ the ingest workflow.
       "event.$":"$",
       "task_config": {
         "excludeFileTypes": "{$.meta.collection.meta.excludeFileTypes}",
-        "multipart_chunksize_mb": "{$.meta.collection.meta.multipart_chunksize_mb}",
+        "s3MultipartChunksizeMb": "{$.meta.collection.meta.s3MultipartChunksizeMb"},
```

Comment:
slight typo, change the value to `"{$.meta.collection.s3MultipartChunksizeMb}"`

Reply:
Pretty sure that second meta is actually correct... That's how Cumulus accesses that value, anyway.

Reply:
Sorry, copy paste error on my part. Need to fix the `"`. Should be `"{$.meta.collection.meta.s3MultipartChunksizeMb}"`,

Reply:
Gah. Fixed.